frontend/add-client/form-inputs/contact-information-inputs.component.tsx

```diff
@@ -149,8 +149,8 @@ export default React.forwardRef(function ContactInformationInputs(
     const data: ContactInformationFormClient = {
       phone,
       smsConsent,
-      streetAddress: capitalize(streetAddress.trim()),
-      city: capitalize(streetAddress.trim()),
+      streetAddress: capitalizeAll(streetAddress.trim()),
```

Comment:
We want each word in the street address to be capitalized, not just the first word.